doc/developers/maintainer.rst

```diff
@@ -109,17 +114,20 @@ Making a release
    the RC releases. These changes should be made in master and cherry-picked
    into the release branch, only before the final release.
 
-   - Edit the doc/whats_new.rst file to add release title and commit
-     statistics. You can retrieve commit statistics with:
+   - Edit the ``doc/v0.99/whats_new.rst`` file to add release title and list of
+     contributors.
+     You can retrieve the list of contributor names with:
 
      ::
 
-       $ git shortlog -s 0.99.33.. | cut -f2- | sort --ignore-case | tr '\n' ';' | sed 's/;/, /g;s/, $//'
+       $ git shortlog -s 0.98.33.. | cut -f2- | sort --ignore-case | tr '\n' ';' | sed 's/;/, /g;s/, $//'
```

Comment:
@ogrisel as I'm doing this right now may I propose this small change?
```suggestion
       $ git shortlog -s 0.98.33.. | cut -f2- | sort --ignore-case | tr '\n' ';' | sed 's/;/, /g;s/, $//' | fold -s
```
This allow to format the output with `n_char <= 80` and breaking at spaces